parts/windows/windowscontainerdfunc.ps1

```diff
@@ -160,17 +160,19 @@ function Install-Containerd {
     $zipfile = [Io.path]::Combine($ENV:TEMP, "containerd.zip")
     DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $zipfile
     Expand-Archive -path $zipfile -DestinationPath $global:ContainerdInstallLocation -Force
-    del $zipfile
+    Remove-Item -Path $zipfile -Force
+
   }
   elseif ($ContainerdUrl.endswith(".tar.gz")) {
     # upstream containerd package is a tar 
     $tarfile = [Io.path]::Combine($ENV:TEMP, "containerd.tar.gz")
     DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $tarfile
     Create-Directory -FullPath $global:ContainerdInstallLocation -DirectoryUsage "storing containerd"
     tar -xzf $tarfile -C $global:ContainerdInstallLocation
-    mv -Force $global:ContainerdInstallLocation\bin\* $global:ContainerdInstallLocation\
-    del $tarfile
-    del -Recurse -Force $global:ContainerdInstallLocation\bin
+
+    Get-ChildItem -Path $global:ContainerdInstallLocation\bin -Recurse -File | Move-Item -Destination $installDir

```

Comment:
Please be careful. They are not equal. mv will move all files under bin\* to dest with keeping the folder structure in bin.
You can create a folder test under bin and put some files in the folder test to run these two commands.
mv will move the folder test to the dest but Get-ChildItem|Move-Item will move files under the folder test to the dest but do not create the folder test in the dest. So if there are some files in different folders with the same name, new command will have the issue.

Reply:
cc @mainred

Reply:
The binaries so far have no conflicted files, and shouldn't have otherwise they should be included in the env path.
But I don't mind use mv instead to avoid any possible accident.